raiden-ts/docs-source/SDK-Development.md

```diff
@@ -193,22 +194,22 @@ Reducers are one of the simplest parts to write. Just some heads-up:
 
 ## Epics
 
 Epics can choose to act on any action or state change, or even dependencies, but it's important they try to follow the UNIX philosophy: do one thing, and do it well. So, as much as possible, each epic should listen a single event type, and output only the minimal action set as outcome of its specific task. The `action$` input observable should be declared as `Observable<RaidenAction>`, as every action goes through (although filtered as early as possible), but try to declare the output as specifically as possible, with a tagged union of any possible action output type.
 
-### Hints, tips & tricks and pits to keep an eye on developing epics:
+### Hints, tips & tricks and pitfalls when developing epics
 
 - Be careful to not complete the output observable if the system is still running, or its function will be lost, as subscriptions aren't restarted.
-- Any unhandled exception will cause a `raidenShutdown` action with the exception to go through as last action, all epics being unsubscribed, action$ and state$ completing and the whole system to shutdown. Handle any expected exception and output a respective `fail` action if relevant, or logging it and swallowing it with `EMPTY` or `ignoreElements`. Unhandled exceptions shouldn't happen in normal operation.
+- Any unhandled exception will cause a `raidenShutdown` action with the exception to go through as last action, all epics being unsubscribed, `action$` and `state$` completing and the whole system to shutdown. Handle any expected exception and output a respective `fail` action if relevant, or logging it and swallowing it with `EMPTY` or `ignoreElements`. Unhandled exceptions shouldn't happen in normal operation.
 - If an epic acts directly (like a map) on `action$`, take care to filter early on the specific action you listen to and output a different action, or else your action output will feedback on your epic and cause an infinite loop. Same if you depend on a `state$` change and your output action causes the same state change that just went through.
 - A common epic pattern: `=> action$.pipe(filter(isActionOf(action)), withLatestFrom(state$), map((action, state) => ...))`
 - Never subscribe inside an epic if not strictly necessary; maps and pipes help into getting a proper _action pipeline_ where a single subscription is used for all epics, making the system more deterministic, declarative and performant.
 - Careful if you use `withLatestFrom` with `action$` or `state$` inside a `mergeMap`, `concatMap`, `exhaustMap`, etc, as the inner (returned) observable is created only when the outer value flows through and the callback of these operators is called. `withLatestFrom` only starts "seeing" values of the `input$` after it's created **and** subscribed, and will discard any source value while the `input$` didn't fire at least once, meaning it can be a silent source of bugs when used inside these mapping operators. e.g. of problematic logic:
```

Comment:
Maybe it's worth adding a paragraph about `deps.latest$`, which is a `ReplaySubject(1)` holding the current/latest emition of several reactive values, and it's safe to use `withLatestFrom` (since they always have something to emit at subscription-time).